OtripleS.Web.Api/Models/Classrooms/Exceptions/FailedClassroomStorageException.cs

```diff
@@ -0,0 +1,18 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System;
+using Xeptions;
+
+namespace OtripleS.Web.Api.Models.Classrooms.Exceptions
+{
+    public class FailedClassroomStorageException : Xeption
+    {
+        public FailedClassroomStorageException(Exception innerException)
+            : base(message: "Failed classroom storage error occurred, contact support.", innerException)
+
```

Comment:
Remove this empty line

Reply:
thanks for the review! extra line removed